Resources/config/doctrine/JobLog.orm.yml

```diff
@@ -0,0 +1,37 @@
+Markup\JobQueueBundle\Entity\JobLog:
+  type: entity
+  table: job_log
+  id:
+    uuid:
+      type: string
+      unique: true
+      length: 100
+  fields:
+    command:
+      type: string
+      length: 2048
+    topic:
+      type: string
+      length: 60
+    added:
+      type: datetime
+    started:
+      type: datetime
+      nullable: true
+    completed:
+      type: datetime
+      nullable: true
+    status:
+      type: string
+      length: 60
+    output:
+      type: string
+      length: 2048
+    peakMemoryUse:
+      type: integer
+      length: 255
+      nullable: true
```

Comment:
How much come peak memory is nullable, do we not always have a reading for the memory?

Reply:
We log jobs which are in pending state as well. They don't have any memory logged before they are executed